Library/Formula/gnu-barcode.rb

```diff
@@ -1,14 +1,18 @@
 class GnuBarcode < Formula
   desc "Convert text strings to printed bars"
   homepage "https://www.gnu.org/software/barcode/"
-  url "http://ftpmirror.gnu.org/barcode/barcode-0.98.tar.gz"
-  mirror "https://ftp.gnu.org/gnu/barcode/barcode-0.98.tar.gz"
-  sha256 "4229e19279b9787ac7e98852fa0bfd93986dce93b9cb07d93a017d68d409b635"
+  url "http://ftpmirror.gnu.org/barcode/barcode-0.99.tar.gz"
+  mirror "https://ftp.gnu.org/gnu/barcode/barcode-0.99.tar.gz"
+  sha256 "7c031cf3eb811242f53664379aebbdd9fae0b7b26b5e5d584c31a9f338154b64"
+
+  patch :DATA
```

Comment:
Can you add a commented out link above this to the upstream bug/patch submission/discussion/etc, Thanks!